src/test/kotlin/no/nav/syfo/melding/api/MeldingApiSpek.kt

```diff
@@ -252,6 +251,113 @@ class MeldingApiSpek : Spek({
+                            dialogmeldingBestillingDTO.dialogmeldingTekst shouldBeEqualTo paminnelseDTO.document.serialize()
+                            dialogmeldingBestillingDTO.dialogmeldingType shouldBeEqualTo DialogmeldingType.DIALOG_FORESPORSEL.name
+                            dialogmeldingBestillingDTO.dialogmeldingKode shouldBeEqualTo DialogmeldingKode.PAMINNELSE_FORESPORSEL.value
+                            dialogmeldingBestillingDTO.dialogmeldingKodeverk shouldBeEqualTo DialogmeldingKodeverk.FORESPORSEL.name
+                            dialogmeldingBestillingDTO.dialogmeldingUuid shouldBeEqualTo pMelding.uuid.toString()
+                            dialogmeldingBestillingDTO.personIdent shouldBeEqualTo pMelding.arbeidstakerPersonIdent
+                            dialogmeldingBestillingDTO.dialogmeldingRefConversation shouldBeEqualTo pMelding.conversationRef.toString()
+                            dialogmeldingBestillingDTO.dialogmeldingRefParent shouldBeEqualTo pMelding.parentRef.toString()
+                            dialogmeldingBestillingDTO.dialogmeldingVedlegg shouldNotBeEqualTo null
+                        }
+                    }
+                }
+                describe("Unnhappy path") {
+                    it("Returns status Unauthorized if no token is supplied") {
+                        testMissingToken(paminnelseApiUrl, HttpMethod.Post)
+                    }
+                    it("returns status Forbidden if denied access to person") {
+                        testDeniedPersonAccess(paminnelseApiUrl, validToken, HttpMethod.Post)
+                    }
+                    it("returns status BadRequest if no $NAV_PERSONIDENT_HEADER is supplied") {
+                        testMissingPersonIdent(paminnelseApiUrl, validToken, HttpMethod.Post)
+                    }
+                    it("returns status BadRequest if $NAV_PERSONIDENT_HEADER with invalid PersonIdent is supplied") {
+                        testInvalidPersonIdent(paminnelseApiUrl, validToken, HttpMethod.Post)
+                    }
+                    it("returns status BadRequest if no melding exists for given uuid") {
+                        with(
+                            handleRequest(HttpMethod.Post, paminnelseApiUrl) {
+                                addHeader(HttpHeaders.Authorization, bearerHeader(validToken))
+                                addHeader(
+                                    NAV_PERSONIDENT_HEADER,
+                                    UserConstants.ARBEIDSTAKER_PERSONIDENT.value
+                                )
+                            }
+                        ) {
+                            response.status() shouldBeEqualTo HttpStatusCode.BadRequest
+                        }
+                    }
+                    it("returns status BadRequest if melding fra behandler exists for given uuid") {
```

Comment:
Kunne man endret ordlyden til feks `returns status BadRequest if meldingUuid is meldingFraBehandler`? For nå tolket jeg det som at man håndterte en sjekk på at en meldingTilBehandler hadde fått et svar likevel, men fant ikke den funksjonaliteten i koden 🤷🏼‍♂️

Det gjorde at jeg kom på: Det vil vel være en case der vi oppretter en oppgave, men behandleren svarer i mellomtiden (før veileder har sett oppgaven og sendt ut påminnelsen). Vi må kanskje også håndtere en slik visning i frontend, som "nuller ut" at det skal sendes en purring? 🤔 Selv om det er snevert case og kanskje ikke noe vi burde bruke så mye tid på?

Reply:
Ja, kan endre den 👍 

Kanskje caset burde håndteres av `ispersonoppgave`? Dvs lukke evt påminnelse-oppgave når det kommer et svar (den vet jo om meldinger fra behandler og lager oppgaver for disse også).